api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -67,6 +74,18 @@ public void setUserDAO(UserDAO dao) {
 		this.dao = dao;
 	}
 	
+	
+	/**
+	 * @return the validTime for which the password reset activation key will be valid
+	 */
+	private int getValidTime() {
+		String contexTime = Context.getAdministrationService()
```

Comment:
typo here, besides I thought this is validTime, why did you rename it to contextTime which doesn't make sense

Reply:
Did you see this comment?